### ISSUE TITLE
Aqara devices lost the configured information in homekit

### ISSUE THREAD

**Issue by JunlongZhao** [snOOrz/homebridge-aqara#22]
Sometimes, some device‘s information restore to default settings in homekit.

![copy of img_4052](https://cloud.githubusercontent.com/assets/8947285/21287885/4c1fae60-c4b0-11e6-9421-fb853ecbdc37.png)

System version: iOS 10.2

**Comment by snOOrz** [comment]
I also found this issue when I use raspberry pi. After I switch to mac mini the issue is gone. I guess there’s some bug in Linux’s mdns implementation.

On 17 Dec 2016, 23:27 +0800, JunlongZhao <notifications@github.com>, wrote:

Sometimes, some device‘s information restore to default settings in homekit.

[copy of img_4052]<https://cloud.githubusercontent.com/assets/8947285/21287885/4c1fae60-c4b0-11e6-9421-fb853ecbdc37.png>

System version: iOS 10.2

**Comment by JunlongZhao** [comment]
Yes, I found this issue when I use Debian and raspberry pi also.
Can it be improved by reducing the amount of Zigbee equipment, or adding another lvmi gateway?

**Comment by snOOrz** [comment]
I don't think so. It's not ZigBee issue. If this happens very often, I'll write an iOS app to fix this issue by one-click.

**Comment by JunlongZhao** [comment]
Thank you so much, I'm almost under orders!
Looking forward to the iOS APP! 😀

**Comment by fxdpz** [comment]
i am using pi and ios 10.3.1. motion sensor and light switch will revert to default at night, the temperature sensor is ok, not sure why.

**Comment by a900927** [comment]
when i see @snOOrz say:" I'll write an iOS app to fix this issue by one-click"
i think the developer is amazing:$
